onnxruntime/core/providers/rknpu/node_attr_helper.cc

```diff
@@ -0,0 +1,116 @@
+//
+// Created by daquexian on 8/3/18.
+//
+
+#include "node_attr_helper.h"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+using std::string;
+using std::vector;
```

Comment:
We always qualify 3rd party names std::string, std;;vector

Reply:
it has been removed.